src/main/java/merge_ats_client/ApiClient.java

```diff
@@ -125,7 +125,7 @@ private void init() {
         json = new JSON();
 
         // Set default User-Agent.
-        setUserAgent("OpenAPI-Generator/1.0.0/java");
+        setUserAgent("OpenAPI-Generator/1.0.0-SNAPSHOT/java");
```

Comment:
These are coming from here: https://github.com/merge-api/merge-backend/blob/develop/openapi/config-merge-ats-java.json#L2

I'm not such a fan of `1.0.0-SNAPSHOT` since this is a minor version release (as you point out in the `README.md` changes). Can you retry your openapi generator with `-c openapi/config-merge-ats-java.json` but modifying that config file to have `1.1.0` there?

Reply:
Huh, thought I had fixed all these. Replaced with proper version